modules/core/src/main/java/org/apache/ignite/internal/processors/cache/GridCachePartitionExchangeManager.java

```diff
@@ -476,6 +489,19 @@ else if (m instanceof GridDhtPartitionDemandLegacyMessage) {
                 });
             }
         }
+
+        MetricRegistry mreg = cctx.kernalContext().metric().registry(PME_METRICS);
+
+        mreg.register(CURRENT_PME_DURATION,
+            () -> getCurrentPMEDuration(false),
+            "Current PME duration in milliseconds.");
+
+        mreg.register(CURRENT_PME_CACHE_OPERATIONS_BLOCKED_DURATION,
```

Comment:
Lest rename to `PME_OPS_BLOCKED_DURATION`

Reply:
@nizhikov What do you think about `CURRENT_PME_OPS_BLOCKED_DURATION`? To avoid confusion with histograms.